gtfs_realtime_translators/translators/mbta.py

```diff
@@ -47,3 +55,6 @@ def __make_trip_updates(cls, predictions):
           trip_updates.append(trip_update)
       
       return trip_updates
+
+    def should_capture_prediction(raw_arrival_time, raw_departure_time):
+      return raw_arrival_time or raw_departure_time
```

Comment:
Readability: the return statement could be a little hard to read, consider abstracting the expression into a variable, but non blocking nonetheless

Reply:
Agreed

Reply:
Are you both suggesting to do this? To me, adding that variable is redundant since we're only going to be using it in the return statement. Doesn't really matter though, and since both of you agree it could be hard to read, I'll change it real quick. Thanks for the input.

```
def should_capture_prediction(raw_arrival_time, raw_departure_time):
      should_capture = raw_arrival_time or raw_departure_time
      return should_capture
      
```